app/create_file.py

```diff
@@ -1 +1,56 @@
-# write your code here
+from datetime import datetime
```

Comment:
you can simplify the module by breaking the logic into atomic functions, this will be more concise, readable and there will be no unnecessary duplication of code. Add the necessary constants. For example - flags and tags to reuse static variables.